test/integration/fixtures/config.js

```diff
@@ -1,30 +1,48 @@
-if (typeof process.env.DC_ADDR !== 'string') {
-  console.error('Missing DC_ADDR environment variable!')
-  process.exit(1)
-}
-
-if (typeof process.env.DC_MAIL_PW !== 'string') {
-  console.error('Missing DC_MAIL_PW environment variable!')
-  process.exit(1)
-}
+const conf = require('rc')('DC', {})
 
-const getConfig = () => {
-  return {
-    notifications: true,
-    enterKeySends: true,
-    showNotificationContent: true,
-    locale: 'en',
+if (conf.account1 === undefined || conf.account1.credentials === undefined) {
+  if (typeof process.env.DC_ADDR !== 'string') {
+    console.error('Missing DC_ADDR environment variable!')
+    process.exit(1)
+  }
+  if (typeof process.env.DC_MAIL_PW !== 'string') {
+    console.error('Missing DC_MAIL_PW environment variable!')
+    process.exit(1)
+  }
+  if (typeof process.env.DC_MAIL_PW1 !== 'string') {
+    console.error('Missing DC_MAIL_PW1 environment variable!')
+    process.exit(1)
+  }
+  if (typeof process.env.DC_MAIL_PW2 !== 'string') {
+    console.error('Missing DC_MAIL_PW2 environment variable!')
+    process.exit(1)
+  }
+  conf.account1 = {
+    credentials: {
```

Comment:
Can you move these values into an extra test scope, because they aren't being used by the app (except for addr - that var is used to determine the last active account)
(aka please don't use config.credentials scope to avoid confusion)